### PR TITLE
Add note about operator case in Sensu Enterprise Dashbaord Collections guide

### PR DESCRIPTION
**Pull request by rgeniesse**
## Description
Adds a note to collections guide that operators must be uppercase.

## Motivation and Context
Clear up confusion when writing search queries to be save as collections and what is supported.

## Review Instructions
Make sure note renders 👌 locally.

### CODE REVIEW

File: content/sensu-enterprise-dashboard/2.12/collections.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ The familiar operators, `AND`, `OR`, and `NOT`, are supported. You may use
 multiple operators within a single query. Be aware that the use of many
 operators may cause higher than normal resource usage.
 
+_NOTE: When using Operators, they must have all letters capitalized. Operators with any lowercase letter are not supported._
```
Comment (cwjohnston) [inline content/sensu-enterprise-dashboard/2.12/collections.md]:
```suggestion
_NOTE: When using operators, all letters must be capitalized. Operators in lowercase letters are not supported._
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
I have suggested a change inline.

**Comment by rgeniesse** [comment]
@cwjohnston all set here I think.

**Review by palourde — APPROVED** [review]
Thank you so much, I had this in my todo list!

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**